projects/RabbitMQ.Client/client/impl/BasicProperties.cs

```diff
@@ -115,7 +115,7 @@ public bool Persistent
         /// </summary>
         public PublicationAddress ReplyToAddress
         {
-            get { return PublicationAddress.Parse(ReplyTo); }
+            get { return PublicationAddress.TryParse(ReplyTo); }
```

Comment:
I think the xml comments on `BasicProperties.ReplyToAddress` and `IBasicProperties.ReplyToAddress' should be updated, too